packages/reporter/cypress/e2e/tests.cy.ts

```diff
@@ -5,14 +5,14 @@ import { MobxRunnerStore } from '@packages/app/src/store/mobx-runner-store'
 let runner: EventEmitter
 let runnables: RootRunnable
 
-function visitAndRenderReporter (runnerStore: MobxRunnerStore = new MobxRunnerStore('e2e')) {
+function visitAndRenderReporter (studioEnabled: boolean = false, studioActive: boolean = false) {
```

Comment:
Updated this because we were only testing when studio was enabled, and these change quite a bit between studio enabled vs studio disabled.